game/characters.py

```diff
@@ -1,17 +1,31 @@
+class Character:
+    Character = []
```

Comment:
Jakej to tu ma ucel?

Reply:
No asi žádný zatím, stačí teda jen u Player a Monster, tak tento list smažu..

Reply:
Spis jsem myslel, ze ani tam se to vlastne k nicemu nepouziva, je mozne ze ses inspiroval tou moji hrou, ale ja bych to dneska uz tak neudelal, tak to muzeme pak nekdy probrat :)